.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        go: [1.12, 1.13]
+    name: Build & Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go ${{ matrix.go }}
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+    - name: Check out
+      uses: actions/checkout@v2
+    - name: Build
+      run: go build -v ./...
+    - name: Test
+      run: go test -v -short ./...
```

Comment:
Looks like there several tests that'll be excluded. What's the plan for when those will run?

Reply:
None as of yet. This mostly mimics what our current Travis CI does.

Reply:
I agree that we should run the full tests... somewhere. I'll make a separate PR to see if there is pain involved in making it work. In my (warm) dev environment, those tests run in ~4 minutes.

```
~/code/bigslice ci ubuntu@ip-10-0-8-137 21s
❯ time go test ./...
ok  	github.com/grailbio/bigslice	232.026s
ok  	github.com/grailbio/bigslice/archive/tarslice	0.068s
?   	github.com/grailbio/bigslice/cmd/badfuncs	[no test files]
?   	github.com/grailbio/bigslice/cmd/bigslice	[no test files]
?   	github.com/grailbio/bigslice/cmd/bigslice/bigslicecmd	[no test files]
?   	github.com/grailbio/bigslice/cmd/slicer	[no test files]
ok  	github.com/grailbio/bigslice/cmd/slicetrace	0.154s
?   	github.com/grailbio/bigslice/cmd/urls	[no test files]
ok  	github.com/grailbio/bigslice/exec	94.445s
ok  	github.com/grailbio/bigslice/frame	0.476s
?   	github.com/grailbio/bigslice/internal/defaultsize	[no test files]
?   	github.com/grailbio/bigslice/internal/slicecache	[no test files]
?   	github.com/grailbio/bigslice/internal/trace	[no test files]
ok  	github.com/grailbio/bigslice/internal/walker	0.023s
ok  	github.com/grailbio/bigslice/internal/zero	0.032s
ok  	github.com/grailbio/bigslice/metrics	0.012s
?   	github.com/grailbio/bigslice/sliceconfig	[no test files]
ok  	github.com/grailbio/bigslice/slicefunc	0.035s
ok  	github.com/grailbio/bigslice/sliceio	3.362s
ok  	github.com/grailbio/bigslice/slicetest	0.033s
ok  	github.com/grailbio/bigslice/slicetype	0.052s
ok  	github.com/grailbio/bigslice/sortio	10.155s
ok  	github.com/grailbio/bigslice/stats	0.002s
ok  	github.com/grailbio/bigslice/typecheck	0.002s
go test ./...  97.18s user 15.14s system 47% cpu 3:54.42 total
```